### ISSUE TITLE
bot doesn't  join to the channel

### ISSUE THREAD

**Issue by andresfmj** [ehamberg/simple-gitbot#1]
Hey, first at all, thank you for sharing your script. I've notice that connects well to the network but doesn't join to the channel automatically. What could it be?

**Comment by ehamberg** [comment]
Hi,

Could it be because the port is blocked? Blocking ports 6665–6667 is quite common. You could try port 8000 which is another port the freenode servers (if that's the network you're using) are listening on.

To test that you're able to connect, you can use the `telnet` command:

    $ telnet chat.freenode.net 8000

If it's able to connect you should then see something like `:wilhelm.freenode.net NOTICE * :*** Looking up your hostname...`.

**Comment by andresfmj** [comment]
Yeah, bot's connecting to the network correctly, but it doesn't join to a channel, I can see it online in the network with a /whois command. 
I thought that could be a delay it'd have, but not, bot never joins to the channel previously defined in the script

**Comment by andresfmj** [comment]
The problem I'm seeing is `say "JOIN ##{$irc_channel}"` being executed before bot connects to the network, and obviously in order to get joined a channel, it is necessary be connected to a network previously, so I don't know exactly how to make the bot get joined after motd message of the network ends displaying.
